api/features/workflows/core/models.py

```diff
@@ -165,6 +165,10 @@ def _publish_environment_feature_versions(
                     },
                     delay_until=environment_feature_version.live_from,
                 )
+                rebuild_environment_document.delay(
+                    kwargs={"environment_id": self.environment_id},
+                    delay_until=environment_feature_version.live_from,
+                )
```

Comment:
Just to confirm: The `live_from` of the environment feature version could be well into the future, correct? But that's ok since our task system is designed to handle it? Just confirming we'd survive, say, a task scheduled further in the future to survive a backend deployment or similar.